gitlab-pipeline/stage/release.yml

```diff
@@ -317,6 +317,7 @@ release_board_artifacts:raspberrypi4:automatic:
   stage: release
   image: debian:buster
   before_script:
+    - *publish_helper_functions
```

Comment:
crap! sorry about that, I was literally running every line of this code, stripping yamls, and mocking commands, but I forgot to add the definitions! sorry. it was next to impossible to test.